NAMESPACE

```diff
@@ -1,12 +1,32 @@
 # Generated by roxygen2: do not edit by hand
 
 S3method(as.list,mts)
+export(BF_Binarize_mean)
+export(BF_iszscored)
+export(CO_Embed2_Basic_tau_incircle)
+export(CO_FirstMin_ac)
+export(CO_trev_1_num)
+export(DN_HistogramMode)
+export(DN_OutlierInclude_abs_001_mdrmd)
+export(EN_SampEn_5_03_sampen1)
+export(FC_LocalSimple_taures)
+export(PH_Walker_prop_01_sw_propcross)
+export(SB_MotifTwo_mean_hhh)
+export(SC_FluctAnal_2_rsrangefit_50_1_logi_prop_r1)
+export(SY_SpreadRandomLocal_100_meantaul)
+export(SY_StdNthDer_1)
 export(acf_features)
 export(arch_stat)
+export(compengine)
 export(crossing_points)
 export(embedding)
 export(entropy)
 export(flat_spots)
+export(hctsa_autocorr)
```

Comment:
We don't need to preface the function names with `hctsa`. Just including a reference in the help file is enough.